.castor/utils.php

```diff
@@ -83,7 +83,12 @@ function create_default_context(): Context
         $data['docker_compose_files'][] = 'docker-compose.override.yml';
     }
 
-    $data['composer_cache_dir'] = cache('composer_cache_dir', fn () => capture(['composer', 'global', 'config', 'cache-dir', '-q'], onFailure: sys_get_temp_dir() . '/castor/composer'));
+    $composerCacheDir = cache('composer_cache_dir', fn () => capture(['composer', 'global', 'config', 'cache-dir', '-q'], onFailure: ''));
+    // If PHP is broken, the output will not be a valid path but an error message
+    if (!is_dir($composerCacheDir)) {
+        $composerCacheDir = sys_get_temp_dir() . '/castor/composer';
```

Comment:
Instead of keeping forever a broken cache, shouldn't we add a way to reset the cache in castor directly?